src/virtualenv/seed/embed/pip_invoke.py

```diff
@@ -39,11 +40,8 @@ def get_pip_install_cmd(self, exe, version):
         pkg_versions = self.package_version()
         for key, ver in pkg_versions.items():
             cmd.append("{}{}".format(key, "=={}".format(ver) if ver is not None else ""))
-        with ExitStack() as stack:
-            folders = set()
-            for context in (ensure_file_on_disk(get_bundled_wheel(p, version), self.app_data) for p in pkg_versions):
-                folders.add(stack.enter_context(context).parent)
-            for folder in folders:
-                cmd.extend(["--find-links", str(folder)])
-                cmd.extend(self.extra_search_dir)
-            yield cmd
+
+        folders = set()
```

Comment:
This is unused?

Reply:
Yes, ExistStack is not used when we don't take folders from `get_bundled_wheel`. Also, the original code is buggy, see https://github.com/pypa/virtualenv/pull/1835

Reply:
Wouldn't be more appropriate to patch the path where the bundle path exists rather than injecting within extra search directory?

Reply:
Well, I am basically following a patch we have for the previous version where it's done in a similar way. We try not to use the original functions for bundled wheels to be one hundred percent sure that we use the correct set of wheels.
Another thing is that we would love to cooperate with upstream, make our changes there somehow, and then drop downstream patches. What we basically need is to search for wheels in `/usr/share/python-wheels` and `/usr/lib64/pythonX.Y/ensurepip/_bundled` (if exists) where `pythonX.Y` is the one virtualenv creates a venv for.

After two days with virtualenv codebase, I think that we can find a working solution which could be merged to upstream.